### DISCUSSION TITLE
Kucoin

### DISCUSSION THREAD

**Discussion by Greensize** [taoteh1221/Slideshow_Crypto_Ticker#6] [Ideas]
Can you add Kucoin to the list?

**Comment by taoteh1221** [comment]
Yes, I can add kucoin to the ticker. It looks like they support the websockets API the app needs here: https://docs.kucoin.com/#symbol-ticker

I'll let you know when it's been added, probably by this coming weekend or earlier. Thanks for the feedback! :)

**Comment by taoteh1221** [comment]
Kucoin integration may take awhile, as I just discovered even public data requests require sending POST data to https://api.kucoin.com/api/v1/bullet-public to receive a temporary public API key. Since we're using javascript only in this app, that's not possible. I need to build in support for users adding their kucoin API key in config.js. I may have time this week to do the additional work, will post back here by next weekend.

**Comment by taoteh1221** [comment]
Kucoin support enabled, WITHOUT needing to use your account API key: https://github.com/taoteh1221/Slideshow_Crypto_Ticker/releases

Enjoy! :)

**Reply by Greensize** [reply]
You rock! Thanks :)

**Reply by taoteh1221** [reply]
Your welcome! :)

**Comment by Greensize** [comment]
Thanks a lot!

Did you have a BNB address?(for donation…)?

**Reply by taoteh1221** [reply]
Thanks for considering a donation to support further development! :) I have BTC and ETH / ERC-20 donation addresses listed below...they are also in README.txt file in the app...

Bitcoin: 3Nw6cvSgnLEFmQ1V4e8RSBG23G7pDjF3hW

ETH / ERC-20: 0x644343e8D0A4cF33eee3E54fE5d5B8BFD0285EF8

**Comment by Greensize** [comment]
Oups…I have a problem…it’s only a black screen.
With the new Config.js

**Reply by taoteh1221** [reply]
Run this command in a terminal login to do a clean upgrade, let me know how it goes:

`
wget -O TICKER-INSTALL.bash https://git.io/Jqzjk;chmod +x TICKER-INSTALL.bash;sudo ./TICKER-INSTALL.bash
`

**Comment by Greensize** [comment]
No it’s black…
Did you have my picture with the error?

**Reply by taoteh1221** [reply]
Did you run the command I sent you? This should correct any manual install errors.

**Reply by taoteh1221** [reply]
No, I don't see a screenshot. Go to the discussion page on github and upload it. First we need that command ran that I sent you, then we can rule out incorrect manual upgrading as the issue. I ran this command on my old install, and it upgraded fine.

**Comment by Greensize** [comment]
There is no  'cache/js/kucoin-auth.js’  folder and file

**Reply by taoteh1221** [reply]
Sounds like you installed manually, an missed some steps. I did update the README.txt file yesterday AFTER packaging the new release, so make sure to check it out if doing a manual install:

https://raw.githubusercontent.com/taoteh1221/Slideshow_Crypto_Ticker/main/README.txt

**Comment by Greensize** [comment]
I left a comment in « Issue"
